src/runtime_src/core/tools/xbutil2/XBDatabase.cpp

```diff
@@ -20,17 +20,50 @@
 #include "tools/common/XBUtilities.h"
 namespace XBU = XBUtilities;
 #include "common/device_core.h"
-
+#include "core/pcie/driver/windows/include/XoclUser_INTF.h"
```

Comment:
Nope.  Not allowed.   This is a windows specific header, XBDatabase is not windows specific.